packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -65,13 +66,13 @@ export function getBlockMoverDescription(
 
 	if ( dir > 0 && ! isLast ) {
 		// Moving down.
-		const movementDirection = getMovementDirection( 'down' );
+		const movementDirection = getMovementDirection( 'down', orientation );
 
 		if ( movementDirection === 'down' ) {
 			return sprintf(
 				// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
 				__(
```

Comment:
I'm not an expert in i18n, but could an alternative option (instead of having separate `getBlockMoverDescription` and `getMultiBlockMoverDescription` functions that repeat the same logic) be to use the `_n` i18n function? (https://developer.wordpress.org/reference/functions/_n/)?

cc @tobifjellner I was wondering if you could advise on whether this fits the best practice for `_n`?

Reply:
We might not be able to use `_n` because the number of  placeholders depends on whether you have multiple blocks selected.

However, we may be able to integrate the logic of `getBlockMoverDescription` and `getMultiBlockMoverDescription` by using a condition such as `isMultiSelected`:

```javascript
const isMultiSelected = selectedCount > 1;

if ( dir > 0 && ! isLast ) {
	// Moving down.
	const movementDirection = getMovementDirection( 'down', orientation );

	if ( movementDirection === 'down' ) {
		return isMultiSelected
			? sprintf(
					// translators: 1: Number of selected blocks, 2: Position of selected blocks
					__(
						'Move %1$d blocks from position %2$d down by one place'
					),
					selectedCount,
					position
				)
			: sprintf(
					// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
					__(
						'Move %1$s block from position %2$d down to position %3$d'
					),
					type,
					position,
					position + 1
				);
	}
}
```

Reply:
Ah, yes, thought so. I'm not sure the ternary improves readability that much, so lets' not worry about it for now.